alfresco-remote-api-exceptions/src/main/java/eu/xenit/alfresco/client/exception/AlfrescoClientResponseException.java

```diff
@@ -0,0 +1,19 @@
+package eu.xenit.alfresco.client.exception;
+
+public class AlfrescoClientResponseException extends AlfrescoClientException {
```

Comment:
The naming is a bit confusing: alfresco - client response - exception ?
Because it's the server-response in this case:

What about a simple `HttpStatusException` extends from `AlfrescoClientException` ?

Reply:
Sounds good!